server/dockerStart.py

```diff
@@ -0,0 +1,66 @@
+import sys

```

Comment:
Yes this sets up some of the ENV variables that the user can define (host, port, production or not). It also starts the container, ends the container, and cleans up when exited.
Of course its possible to start the containers without this file by exporting the ENV variables, but for future ease of use I think it is necessary.

For now I'm just going to comment out the nginx stuff till we get it live, in case it is needed.
I changed the .env extension to .conf files to avoid confusion. using these files and passing them to the docker-compose is the "docker way" of doing it. It is possible to export the FLASK_ENV and FLASK_DEBUG vars in dockerStart.py thus not needing the conf files, but its standard practice to use the files.

If we end up not needing nginx, docker-compose-prod.yml can be deleted.